src/makeCalculator.js

```diff
@@ -36,8 +36,49 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makecalculator() {
+  const calc = {
```

Comment:
```suggestion
  return {
```
You can return this object right away